eslint/common.yml

```diff
@@ -1,9 +1,6 @@
 # Body Labs JavaScript style. Derived from Crockford style, with a bunch of
 # tweaks.
 
-# Don't inherit settings from e.g. ~.
-root: true
-
```

Comment:
This turns out to be not as necessary as I had thought. According to the docs it skips `~/.eslintrc` when other configs are present. That's the main thing I wanted to prevent against here. Most of us don't lay out our projects inside each other, so I don't think we need to worry about other `~/.eslintrc`'s in parent directories.

If we decide down the line that we want this, we'll have to add it as boilerplate _in each project_.